src/ride/ride.c

```diff
@@ -4306,13 +4309,6 @@ bool ride_create_vehicles(rct_ride *ride, int rideIndex, rct_xy_element *element
  */
 static bool sub_6D31A6(rct_ride *ride, bool isApplying)
```

Comment:
I guess you can probably rename this to validate_cable_lift_hill or something. Its checking and setting track flags, so I guess its doing more than just validating. Perhaps try_initialise_cable_lift_hill or something.